packages/database/src/migrations/20201201082830-AddProjectPalauOlangch-modifies-data.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const COUNTRY_CODE = 'PW';
+const PROJECT_CODE = 'olangch_palau';
+const PROJECT_NAME = 'MoH Project Olangch';
+const DASHBOARDGROUP_NAME = 'General';
+const DASHBOARDGROUP_CODE = 'PW_Olangch';
```

Comment:
According to an implicit existing convention:

```suggestion
const DASHBOARDGROUP_CODE = 'OLANGCH_PALAU_Project';
```

Although I don't know why we use upper case for the project name, `Olangch_Palau_Project` would work just as fine for me, let's stick to the existing convention

Reply:
Ah yes good pick up thanks!